syntax = "proto3";

package fraud;

import "common.proto";

service FraudService {
    rpc SayFraud (common.Request) returns (OrderResponse);
    rpc InitVerification (common.ItemsInitRequest) returns (common.Empty);
}

message OrderResponse {
    bool is_fraud = 1;
    string message = 2;
}
